src/core/amount.js

```diff
@@ -343,12 +358,15 @@ Amount.prototype.invert = function() {
  */
 
 Amount.prototype.canonicalize = function(roundingMode) {
```

Comment:
Does Amount even need a canonicalize method anymore? Don't the value classes already ensure canonicalization?